packages/nextjs/src/common/wrapApiHandlerWithSentryVercelCrons.ts

```diff
@@ -42,7 +49,6 @@ export function wrapApiHandlerWithSentryVercelCrons<F extends (...args: any[]) =
             status: 'in_progress',
           },
           {
-            checkinMargin: 2, // two minutes - in case Vercel has a blip
```

Comment:
Why did we remove this?

Reply:
Had a discussion with the crons team. The missed checkins were a bug in the product when the margin is set to 0.